src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,117 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Book {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
We recommend using GenerationType.SEQUENCE as a strategy because PostgreSQL uses a global incremental id, shared by all tables. This is the strategy recommended by Hibernate 😊